datastore/kafka/kafka.go

```diff
@@ -113,9 +107,14 @@ func (p *Producer) handleProducerEvents(ackChan chan (*telemetry.Record)) {
 		case kafka.Error:
 			p.logError(fmt.Errorf("producer_error %v", ev))
```

Comment:
We should probably add a metrics here to count the kafka errors as seen by the producer. Will help see problems at a glance.

Reply:
log error increments error metrics https://github.com/teslamotors/fleet-telemetry/blob/0431c0fa97158bbac5d2c7fb2eb0b517e2c95530/datastore/kafka/kafka.go#L124